package/yast2-packager.changes

```diff
@@ -2276,7 +2282,7 @@ Fri May 16 15:41:44 CEST 2014 - locilka@suse.com
 -------------------------------------------------------------------
 Fri May 16 12:31:52 UTC 2014 - jreidinger@suse.com
 
-- Fix preffering DVD during installation (bnc#865819,bnc#780617)
+- Fix preferring DVD during installation (bnc#865819,bnc#780617)
```

Comment:
I'm not sure if OBS will accept this change, in the past there was a bot checking that the changes are only added at the top and the history was not changed. But I do not know if it is still used, maybe just try it and see...

Reply:
We will see if fixing typos is impossible.